8.x/theme-development/representative.md

```diff
@@ -0,0 +1,381 @@
+
+```php{6-8}
+// public/views/register-representative.blade.php
+
+<select class="form-control number-control" name="representative_type">
+     <option value="">هیج کدام</option>
+    <option value="مشتریان فعلی مجموعه" id="manual-representative">مشتریان فعلی مجموعه</option>
+         @foreach(representative_get_options() as $option)
+            <option value="{{$option}}">{{$option}}</option>
+        @endforeach
+</select>
+
+```
+
+You can replace a radio button instead of a select/option tag.
+
+```php
+
+@foreach(representative_get_options() as $option)
+<lable for="">
+    {{$option}}
+<input type="radio" class="form-control" name="representative_type"
+  value="{{$option}}">
+ </lable>           
+@endforeach
+
+```
+
+Now deploy the project and see the result :
+```
+./deploy.sh
+```
+![Drop-down](/repDropDown.png)
+
+**Note 1:** Options were already passed in the form manually.
+
+**Note 2:** Consider not changing the "name" attribute.
+
+**Note 3:** You can add CSS to adjust options location on the form.
+
+---
+
+### How to set conditional option
+
+#### representative_is_customer_representative_enabled()
+
+The other function is `representative_is_customer_representative_enabled()`. By using this function you will be able to set a condition in which whether a part should be displayed or not.
+
+The function:
+```php
+function representative_is_customer_representative_enabled(): bool {
+        return \App\Utils\CMS\Setting\Representative\RepresentativeSettingService::isCustomerRepresentativeEnabled();
+    }
+```
+
+In the upcoming section, focus on understanding how to utilize the function and the output.
+
+```php
+//register-representative.blade.php
+
+@if(representative_is_enabled())
+
+    ...
+
+    @if(representative_is_customer_representative_enabled())
+        <div class="col-md-6 col-sm-6 col-xs-12">
+            <div class="form-group">
+                <label for="family">شماره تماس معرف در صورت تمایل</label>
+                <input type="text" class="form-control number-control" name=representative-username"
+                placeholder="شماره تماس معرف" 
+                value="{{old('representative-username')}} " > 
+            </div>
+        </div>
+    @endif
+ @endif
+```
+ So now if you enable the second option of the representative form in your admin pannel, this part will be displayed in the registration form, otherwise the field will be removed.
+
+![Conditional option1](/repcondition.png)
+
+Another example is to build a conditional option to show a specific field when a specific options is selected.
+For example if you set current customers as representative, you will need a field for that customer information(for example, the phone number). So the number input will be displayed only when the current customer is selected. Otherwise this input won't be shown. See its step-by-step instruction:
```

Comment:
Suppose you add a new item named `our current customers` to the `How to get familiar with us` field in the registration form.Your idea is to display the `representative phone number` field into the registration form only when the user selects this new item. Let's see how to implement this idea.